share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+.\"-
+.\" Copyright (c) 2024 M. Warner Losh <imp@FreeBSD.org>
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.Dd July 3, 2024
+.Dt CDEFS 9
+.Os
+.Sh NAME
+.Nm cdefs
+.Nd "Compiler portability definitions"
```

Comment:
```suggestion
.Nd compiler portability macro definitions
```

Reply:
Not Dd; sorry, Nd as in Descriptions shouldn't start with capital letters unless the beginning is a proper noun.

Reply:
Why'd you add 'macro' here?

Reply:
Perhaps I misinterpreted, and macros is the wrong term?

I inturpreted cdefs.h in my head as library of compatability macros for compiler support. Thus I think, people may use apropos and grep in combination with the term macro.

Reply:
Fair point. I'll add it. These are pseudo macros since they sometimes are and sometimes aren't actually a macro.

But it does make sense from a discoverability standpoint